node/node_test.go

```diff
@@ -9,15 +9,13 @@ import (
 	"testing"
 	"time"
 
+	dbm "github.com/line/tm-db/v2"
```

Comment:
Unnecessary changes?

Reply:
It's an effect of the `make format`. Currently, imports in many source files are disjointed, separated by blank lines in meaningless units.

I'll fix this, and I think if the opportunity to correct them in an individual PR isn't given, we should organize them correctly somewhere and make the `make format` and CI behavior the same (`Makefile` seem better than CI at the moment), as a "good first issue".

```
gofmt -w -s
goimports -w -local github.com/line/ostracon
```